src/mod_muc.erl

```diff
@@ -330,9 +292,17 @@ init([Host, Opts]) ->
     ejabberd_router:register_route(MyHost, mongoose_packet_handler:new(?MODULE, State)),
     mongoose_subhosts:register(Host, MyHost),
 
-    load_permanent_rooms(MyHost, Host,
-                         {Access, AccessCreate, AccessAdmin, AccessPersistent},
-                         HistorySize, RoomShaper, HttpAuthPool),
+    case gen_mod:get_module_opt(Host, mod_muc, load_permanent_rooms_at_startup, false) of
+        false ->
+            ?INFO_MSG("issue=load_permanent_rooms_at_startup, skip=true, "
```

Comment:
What's the difference between `issue` and `event` in logs?

Reply:
Ideological issues. Once we choose one, we can replace it everywhere.

```erlang
~/erlang/esl/2017/MongooseIM|feature-muc-backends-revived $ git grep "event=" | wc -l                                             
34                              
~/erlang/esl/2017/MongooseIM|feature-muc-backends-revived $ git grep "issue=" | wc -l                                             
35 
```